currency/currency_test.go

```diff
@@ -0,0 +1,16 @@
+package currency
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+	stripe "github.com/stripe/stripe-go"
+	_ "github.com/stripe/stripe-go/testing"
+)
+
+func TestCurrencyWithChargeParams(t *testing.T) {
+	params := &stripe.ChargeParams{
+		Currency: USD,
+	}
+	assert.Equal(t, USD, params.Currency)
```

Comment:
Admittedly not much. I just put it in there so that there was a single test target in the file. It doesn't matter that much if there isn't though, so removed in 3d4257b.